pkgs/top-level/all-packages.nix

```diff
@@ -215,7 +215,11 @@ in
 
   dotnetCorePackages = recurseIntoAttrs (callPackage ../development/compilers/dotnet {});
 
-  dotnet-sdk = dotnetCorePackages.sdk_2_2;
+  dotnet-sdk = dotnetCorePackages.sdk_2_1;
```

Comment:
:thinking: Some packages might require .NET Core 3.1 while others might still use 2.1. Should we expose those to the top level directly with a version suffix?

Reply:
I think having `dotnet-sdk_2` and `dotnet-sdk_3` top level attrs makes sense

Reply:
Is it better to add top level attributes for each version or to override the attribute for specific packages only ?

Reply:
I would do:
```
dotnet-sdk = dotnetCorePackages.sdk_2_1;

dotnet-sdk_2 = dotnetCorePackages.sdk_2_1;

dotnet-sdk_3 = dotnetCorePackages.sdk_3_1;
```

And just add two commits:
```
dotnet-sdk_2: add alias
dotnet-sdk_3: add alias
```

Reply:
In another PR we can deal with the controversy of bumping dotnet-sdk to 3